web/Controllers/AdminController.cs

```diff
@@ -164,9 +166,10 @@ public ActionResult GetInviteLink(int landlordId)
                 flashMessageBody = "Landlord already has an invite link";
             }
 
-            FlashMessage(_logger, (flashMessageBody, "success", flashMessageBody + $": {(HttpContext.Request.IsHttps ? "https" : "http")}" +
+            _logger.LogInformation(flashMessageBody);
+            AddFlashMessage("success", flashMessageBody + $": {(HttpContext.Request.IsHttps ? "https" : "http")}" +

```

Comment:
could we extract the flashMessage body out of this call to add message and then pass it in, something like 
```suggestion
            _logger.LogInformation(logMessage);
            var flashMessage = logMessage +
                $": {(HttpContext.Request.IsHttps ? "https" : "http")}://{HttpContext.Request.Host}/invite/{inviteLink}";
            AddFlashMessage("success", flashMessage);
```